retriever/try_install_modified.py

```diff
@@ -0,0 +1,139 @@
+"""Attempt to update datasets in all database management systems.
+This module, when run, attempts to install datasets from modified Retriever
+scripts in the /scripts folder (except for those listed in IGNORE), for each
+engine in ENGINE_LIST() from __init__.py. In other words, it trys to install
+using all possible combinations of database platform and script and checks
+to see if there are any errors. It does not check the values in the database.
+"""
+from __future__ import print_function
+from __future__ import absolute_import
+import os
+import sys
+from imp import reload
+from retriever.lib.tools import choose_engine
+from retriever import MODULE_LIST, ENGINE_LIST, SCRIPT_LIST
+from retriever.version import get_module_version, update_version_file
```

Comment:
I see. Adding update_version_file might be beneficial, but I don't think it's absolutely necessary. That would need the approval of @henrykironde though. I've also looked over the code again, and added a few more suggestions which completely circumvent the issues you were facing earlier with this import. It seems to work after those, so test it out and let me know.